src/Features/Core/Portable/ExtractMethod/InsertionPoint.cs

```diff
@@ -16,15 +13,7 @@ internal class InsertionPoint
         private readonly SyntaxAnnotation _annotation;
         private readonly Lazy<SyntaxNode?> _context;
 
-        public static async Task<InsertionPoint> CreateAsync(SemanticDocument document, SyntaxNode node, CancellationToken cancellationToken)
-        {
-            var root = document.Root;
-            var annotation = new SyntaxAnnotation();
-            var newRoot = root.ReplaceNode(node, node.WithAdditionalAnnotations(annotation));
-            return new InsertionPoint(await document.WithSyntaxRootAsync(newRoot, cancellationToken).ConfigureAwait(false), annotation);
-        }
```

Comment:
we now do the rewriting to annotated teh insertion point at the same time that we annotate the variables to track.